sources/api/migration/migrator/src/main.rs

```diff
@@ -82,11 +124,65 @@ fn run() -> Result<()> {
             process::exit(0);
         });
 
-    let migrations = find_migrations(
-        &args.migration_directories,
-        &current_version,
-        &args.migrate_to_version,
-    )?;
+    // DEPRECATED CODE BEGIN ///////////////////////////////////////////////////////////////////////
+    // check if the `from_version` supports signed migrations. if not, run the 'old'
+    // unsigned migrations code and return.
+    if !are_migrations_signed(&current_version) {
+        // note in the system journal that the unsigned code path ran.
+        eprintln!("migrator is running unsigned migrations");
+        warn!("migrator is running unsigned migrations");
+        return find_and_run_unsigned_migrations(
+            &args.migration_directory,
+            &args.datastore_path, // TODO(brigmatt) make sure this is correct
+            &current_version,
+            &args.migrate_to_version,
+            &direction,
+        );
+    } else {
+        // note in the system journal that the signed code path ran.
+        info!("migrator is running signed migrations");
+        eprintln!("migrator is running signed migrations");
```

Comment:
I don't think we need these messages in the logs/journal, we know which type of migrations ran based on the version.  (Ties with my other comments about simplifying/removing the branching.)